src/helpers/api_helper.js

```diff
@@ -7,7 +7,7 @@ import { getLoggedInUser } from "./fakebackend_helper"
 const token = accessToken
 
 //apply base url for axios
-export const API_URL = "https://hackusu-backend-ea481c29b26f.herokuapp.com"
```

Comment:
Todo make this change automatically by what branch is being auto deployed or what code is being built